tracer/src/Datadog.Trace/AppSec/IastRaspVulnerabilityManager.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="IastRaspVulnerabilityManager.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.AppSec.Rasp;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Iast;
+
+#nullable enable
+
+namespace Datadog.Trace.AppSec;
+
+internal static class IastRaspVulnerabilityManager
```

Comment:
I'd call this simply `VulnerabilityManager` or `VulnerabilitiesModule` to match the other modules

Reply:
Done!